scripts/challenge11.js

```diff
@@ -0,0 +1,16 @@
+function cubeOdd(arr) {
+
+  // insert code here >.<
+  for(var i=0; i<arr.length; i++){
```

Comment:
I'd recommend next time for a situation like this to just do your map, filter, and reduce and check if the result is NaN instead of looping through the original array first.

Reply:
https://repl.it/repls/CandidDiligentDrawings